frontend/src/App.tsx

```diff
@@ -1,36 +1,93 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
+import styled from 'styled-components';
 import logo from './logo.svg';
-import './App.css';
- 
-class App extends Component {
- 
-    state:any = {};
- 
-    componentDidMount() {
-        setInterval(this.hello, 250);
+import LandingPage from './views/Landing';
+import fetchHello from './api/hello';
```

Comment:
In my internship, we ordered imports alphabetically -- is that something we should do? That might pertain more to Java anyway, but I wanted to bring this up anyway.

Reply:
Fixed, although I have two notes/questions:

1. ESLint recommends that you have package imports first (e.g. 'react') and then relative imports (e.g. './logo.svg'), so I left them separate but sorted each group. I think can modify ESLint rules though if that's something you think should change.
2. When sorting alphabetically, do you prefer sorting by the thing you're importing (e.g. LandingPage) or by the file it's coming from (e.g. './api/hello')? In this scenario both lead to the same result so it doesn't matter, but in future situations it could be good to clarify (my personal preference is by file but don't feel too strongly about this).

Reply:
1. That sounds good, thanks for the clarification there.
2. I don't have any personal preference, so let's go by file moving forward.